server/src/main/java/com/defold/extender/AsyncBuilder.java

```diff
@@ -97,8 +101,14 @@ public void asyncBuildEngine(MetricsWriter metricsWriter, String platform, Strin
 
             // Resolve Gradle dependencies
             if (platform.contains("android")) {
-                List<File> gradlePackages = extender.resolve(gradleService);
-                metricsWriter.measureGradleDownload(gradlePackages, gradleService.getCacheSize());
+                extender.resolve(gradleService);
+                metricsWriter.measureGradleDownload(gradleService.getCacheSize());
+            }
+
+            // Resolve CocoaPods dependencies
+            if (platform.contains("ios") || platform.contains("osx")) {
+                extender.resolve(cocoaPodsService);
+                metricsWriter.measureCocoaPodsInstallation();
```

Comment:
Does CocoaPods has its own cache?
If yes, can we add it here? If not, should we do something like our own?

Reply:
Yes CocoaPods has a cache: /Users/USER/Library/Caches/CocoaPods/Pods and it nowadays has support for concurrent installs of pods using the same cache location: https://github.com/CocoaPods/CocoaPods/pull/10018